pygmt/base_plotting.py

```diff
@@ -100,34 +101,34 @@ def coast(self, **kwargs):
         ----------
         {J}
         {R}
-        A : int, float, or str
+        area_thresh : int, float, or str
             ``'min_area[/min_level/max_level][+ag|i|s|S][+r|l][+ppercent]'``
             Features with an area smaller than min_area in km^2 or of
             hierarchical level that is lower than min_level or higher than
             max_level will not be plotted.
         {B}
         C : str
```

Comment:
No long alias for `C` in coast? GMT.jl uses `river_fill`, but has a system to handle lakes/rivers separately using a tuple. See https://www.generic-mapping-tools.org/GMT.jl/latest/coast/. We could do `riverlake_fill` :laughing:

Reply:
I'm hesitating to add new long-name aliases in this PR. We should have more discussions in separate issues (i.e., one issue for each module), as we don't want to break backward compatibility due to some stupid names.

Reply:
Ok, I had a feeling it wasn't just randomly left out. Will treat it as an exception now then.